atomics/T1090.001/T1090.001.yaml

```diff
@@ -18,17 +18,29 @@ atomic_tests:
     proxy_port:
       description: Proxy server port
       type: integer
-      default: 8080
+      default: 3128
     proxy_scheme:
       description: Protocol to proxy (http or https)
       type: string
       default: http
+  dependencies:
+  - description: Squid must be installed and running
+    prereq_command: if [ -x "$(command -v squid --version)" ]; then exit 0; else exit 1; fi;
+    get_prereq_command: |-
+      sudo DEBIAN_FRONTEND=noninteractive apt-get -y install squid curl
+      sed -i 's/http_access deny all/#http_access deny all/' /etc/squid/squid.conf
+      echo "running squid"
+      squid -N -d 1 &
   executor:
     command: |
       export #{proxy_scheme}_proxy=#{proxy_server}:#{proxy_port}
+      curl http://google.com
+    elevation_required: true
```

Comment:
`elevation_required` is technically `false` for running the command. Installing prereqs requires elevation but I wont count that one here because it depends upon another variable whether squid must be installed or not.

Reply:
yes, with the "run_squid" flag the install phase won't be executed if it's false, so the elevation isn't required. But, if the flag is true, they will go through the proxy install step and the elevation is needed, right? 
I think we should keep it